lib/matplotlib/rcsetup.py

```diff
@@ -888,6 +872,36 @@ def validate_animation_writer_path(p):
         modules["matplotlib.animation"].writers.set_dirty()
     return p
 
+# A validator dedicated to the named line styles, based on the items in
+# ls_mapper, and a list of possible strings read from Line2D.set_linestyle
+validate_named_linestyle = ValidateInStrings('linestyle',
```

Comment:
Ok I'll do that! (I was hesitating because most of the other validators are public).

Speaking of public API, I think I've done something wrong (:sheep:) by genuinely getting rid off of `validate_negative_linestyle` and `validate_negative_linestyle_legacy` as they were both part of the public API. I should have used some deprecation warning instead, shouldn't I? Or at least just leave them (because if `validate_linestyle` is made private, there will be no alternative)?